mat/matrix.go

```diff
@@ -217,15 +217,16 @@ func untranspose(a Matrix) (Matrix, bool) {
 	return a, false
 }
 
-// untransposeExtract first untransposes the matrix. It then returns the
-// unextracted matrix if it is one of the built-in types, extracts it into one
-// of the built-in types if it implements the corresponding Raw method and
-// meets the conditions, or returns the Matrix otherwise.
+// untransposeExtract returns an untransposed matrix in a built-in matrix type.
+//
+// The untransposed matrix is return unaltered if it is a built-in matrix type.
```

Comment:
Sorry. s/return/returned/, my error.

Reply:
Ah shoot, caught one of them, not the other. Done.